pkg/types/ovirt/platform.go

```diff
@@ -0,0 +1,39 @@
+package ovirt
+
+// Platform stores all the global configuration that all
+// machinesets use.
+type Platform struct {
+	// URL is oVirt's API endpoint, e.g https://ovirt-engine-fqdn/ovirt-engine/api
+	URL string `json:"ovirt_url,omitempty"`
```

Comment:
remove `omitempty` from all required fields in platform

Reply:
done